.storybook/webpack.config.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+
+// load the default config generator.
+const genDefaultConfig = require('@storybook/react/dist/server/config/defaults/webpack.config.js');
+
+module.exports = (baseConfig, env) => {
+  const config = genDefaultConfig(baseConfig, env);
+
+  // config.module.rules.push({
```

Comment:
please don't push commented out code in to master unless there's a comment with it saying why it may be needed later